src/gc.c

```diff
@@ -3575,7 +3588,7 @@ void jl_gc_init(void)
     if (total_mem < 128e9)
         percent = total_mem * 2.34375e-12 + 0.6; // 60% at 0 gigs and 90% at 128 to not
     else                                         // overcommit too much on memory contrained devices
-        percent = 0.9;
+        percent = 0.8;
```

Comment:
I think these thresholds are now provably wrong now. For `uv_get_total_memory`, we want allow allocating 10x that amount before we start worry too much about the OOM killer, otherwise we are just trashing performance by thrashing swap for no reason. For `uv_get_constrained_memory`, the number should be something like 0.8% - 256MB so that we account for the fixed cost of the system image (roughly) and otherwise try to push right up against the limit (leaving only about 20% for other parts of the program, as a guess).